src/wp-includes/class-wp-script-modules.php

```diff
@@ -225,6 +226,34 @@ public function print_import_map() {
 		}
 	}
 
+	/**
+	 * Prints the necessary script to load import map polyfill for browsers that
+	 * do not support import maps.
+	 *
+	 * @since 6.5.0
+	 */
+	public function print_import_map_polyfill() {
+		$import_map = $this->get_import_map();
+		if ( ! empty( $import_map['imports'] ) ) {
+			$test = 'HTMLScriptElement.supports && HTMLScriptElement.supports("importmap")';
+			$src  = add_query_arg( 'ver', get_bloginfo( 'version' ), includes_url( 'js/dist/vendor/wp-polyfill-importmap.min.js' ) );
```

Comment:
```suggestion
			$src  = add_query_arg( 'ver', '1.8.2', includes_url( 'js/dist/vendor/wp-polyfill-importmap.min.js' ) );
```
The package version number is going to be more stable.

Reply:
Done in https://github.com/WordPress/wordpress-develop/pull/5947/commits/4c397a653155ec1f35524a09b70749adbb00368f